frontmacs-editing.el

```diff
@@ -49,15 +49,18 @@
 (require 'undo-tree)
 (global-undo-tree-mode 1)
 
+;; set fringe widths
```

Comment:
I'd put your comment in the PR here: E.g.

;; make the left fringe 2 pixels so the hl-diff indicators aren't so fat
;; leave the right fringe width at the default 8 pixels